psse/psse-converter/src/main/java/com/powsybl/psse/converter/TransformerConverter.java

```diff
@@ -745,6 +745,7 @@ private static boolean addControlThreeWindingsTransformerLeg(Network network, St
 
     private static boolean defineVoltageControl(Network network, String id, PsseTransformerWinding winding, RatioTapChanger rtc,
         boolean regulatingForcedToOff) {
+        // TODO : remove this ?
```

Comment:
I think you could remove this test and only keep the "if (Math.abs(winding.getCod()) != 1)"

Reply:
psse would be done in another PR